teamwork.css

```diff
@@ -33,7 +33,7 @@ body {
   padding: 24px 48px 24px 24px;
 }
 
-p {
+.paragraph {
```

Comment:
duplication of class name in one file . please merge it

Reply:
done

Reply:
nope